source/puppet/5.5/upgrade_major_server.markdown

```diff
@@ -44,14 +44,24 @@ If you have multiple Puppet masters, upgrade or replace the certificate authorit
 
 Repeat the following steps for each Puppet Server until you're running a pure Puppet 5/Puppet Server 5 infrastructure.
 
-### Install the latest Puppet Server
+### Install the latest Puppet Server 2.x
```

Comment:
Can we mention that this is only required if on Puppet Server < 2.latest either in the title or below? (I might be overly concerned that someone will first downgrade to 2.8 from 5.x)

Reply:
@justinstoller Do you mean on Puppet Server 2.0 and earlier or later?